File: android/BeroepsproductBlok4RecipesWorldwide/app/src/main/java/com/nl/recipeapp/model/Religion.java
package com.nl.recipeapp.model;

public class Religion {
    String id, name;

    public Religion(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public Religion() {

    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return name;
    }
}

File: frontend/model/Religion.php
<?php
namespace frontend\model;
require_once(dirname(__FILE__,2) . '/exception/ModelNullException.php');
require_once(dirname(__FILE__,1) . '/Model.php');
require_once(dirname(__FILE__,1) . '/Update.php');
  class Religion extends Model{
    private $name;
    private $id;

    public function __construct(String $name = null, int $id = null){
        $this->name = $name;
        $this->id = $id;
    }
    //the name of the religion
    public function getName() : String{
        return $this->name;
    }
    //the name of the religion
    public function setName(String $name){
        $this->name = $name;
    }
    //the id of the religion
    public function getId() : int{
        return $this->id;
    }
    //the id of the religion
    public function setId(int $id) {
        $this->id = $id;
    }
}
